src/braket/default_simulator/openqasm/parser/openqasm_ast.py

```diff
@@ -0,0 +1,1059 @@
+# Apache V2 usage with no significant modifications
```

Comment:
Is it possible at all to just use the one in the openqasm repo? If not, can you also label any modifications just for clarity?

Reply:
per offline discussion - No, since they don't have the generated files and that requires a java build step